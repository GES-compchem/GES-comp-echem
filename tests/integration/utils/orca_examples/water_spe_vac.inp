%pal
  nprocs 4
end

%maxcore 350

! PBE def2-SVP 
! RIJCOSX def2/J

* xyzfile 0 1 water.xyz
