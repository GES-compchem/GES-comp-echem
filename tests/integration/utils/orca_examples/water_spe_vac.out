
                                 *****************
                                 * O   R   C   A *
                                 *****************

                                            #,                                       
                                            ###                                      
                                            ####                                     
                                            #####                                    
                                            ######                                   
                                           ########,                                 
                                     ,,################,,,,,                         
                               ,,#################################,,                 
                          ,,##########################################,,             
                       ,#########################################, ''#####,          
                    ,#############################################,,   '####,        
                  ,##################################################,,,,####,       
                ,###########''''           ''''###############################       
              ,#####''   ,,,,##########,,,,          '''####'''          '####       
            ,##' ,,,,###########################,,,                        '##       
           ' ,,###''''                  '''############,,,                           
         ,,##''                                '''############,,,,        ,,,,,,###''
      ,#''                                            '''#######################'''  
     '                                                          ''''####''''         
             ,#######,   #######,   ,#######,      ##                                
            ,#'     '#,  ##    ##  ,#'     '#,    #''#        ######   ,####,        
            ##       ##  ##   ,#'  ##            #'  '#       #        #'  '#        
            ##       ##  #######   ##           ,######,      #####,   #    #        
            '#,     ,#'  ##    ##  '#,     ,#' ,#      #,         ##   #,  ,#        
             '#######'   ##     ##  '#######'  #'      '#     #####' # '####'        



                  #######################################################
                  #                        -***-                        #
                  #          Department of theory and spectroscopy      #
                  #    Directorship and core code : Frank Neese         #
                  #        Max Planck Institute fuer Kohlenforschung    #
                  #                Kaiser Wilhelm Platz 1               #
                  #                 D-45470 Muelheim/Ruhr               #
                  #                      Germany                        #
                  #                                                     #
                  #                  All rights reserved                #
                  #                        -***-                        #
                  #######################################################


                         Program Version 5.0.3 -  RELEASE  -


 With contributions from (in alphabetic order):
   Daniel Aravena         : Magnetic Suceptibility
   Michael Atanasov       : Ab Initio Ligand Field Theory (pilot matlab implementation)
   Alexander A. Auer      : GIAO ZORA, VPT2 properties, NMR spectrum
   Ute Becker             : Parallelization
   Giovanni Bistoni       : ED, misc. LED, open-shell LED, HFLD
   Martin Brehm           : Molecular dynamics
   Dmytro Bykov           : SCF Hessian
   Vijay G. Chilkuri      : MRCI spin determinant printing, contributions to CSF-ICE
   Dipayan Datta          : RHF DLPNO-CCSD density
   Achintya Kumar Dutta   : EOM-CC, STEOM-CC
   Dmitry Ganyushin       : Spin-Orbit,Spin-Spin,Magnetic field MRCI
   Miquel Garcia          : C-PCM and meta-GGA Hessian, CC/C-PCM, Gaussian charge scheme
   Yang Guo               : DLPNO-NEVPT2, F12-NEVPT2, CIM, IAO-localization
   Andreas Hansen         : Spin unrestricted coupled pair/coupled cluster methods
   Benjamin Helmich-Paris : MC-RPA, TRAH-SCF, COSX integrals
   Lee Huntington         : MR-EOM, pCC
   Robert Izsak           : Overlap fitted RIJCOSX, COSX-SCS-MP3, EOM
   Marcus Kettner         : VPT2
   Christian Kollmar      : KDIIS, OOCD, Brueckner-CCSD(T), CCSD density, CASPT2, CASPT2-K
   Simone Kossmann        : Meta GGA functionals, TD-DFT gradient, OOMP2, MP2 Hessian
   Martin Krupicka        : Initial AUTO-CI
   Lucas Lang             : DCDCAS
   Marvin Lechner         : AUTO-CI (C++ implementation), FIC-MRCC
   Dagmar Lenk            : GEPOL surface, SMD
   Dimitrios Liakos       : Extrapolation schemes; Compound Job, initial MDCI parallelization
   Dimitrios Manganas     : Further ROCIS development; embedding schemes
   Dimitrios Pantazis     : SARC Basis sets
   Anastasios Papadopoulos: AUTO-CI, single reference methods and gradients
   Taras Petrenko         : DFT Hessian,TD-DFT gradient, ASA, ECA, R-Raman, ABS, FL, XAS/XES, NRVS
   Peter Pinski           : DLPNO-MP2, DLPNO-MP2 Gradient
   Christoph Reimann      : Effective Core Potentials
   Marius Retegan         : Local ZFS, SOC
   Christoph Riplinger    : Optimizer, TS searches, QM/MM, DLPNO-CCSD(T), (RO)-DLPNO pert. Triples
   Tobias Risthaus        : Range-separated hybrids, TD-DFT gradient, RPA, STAB
   Michael Roemelt        : Original ROCIS implementation
   Masaaki Saitow         : Open-shell DLPNO-CCSD energy and density
   Barbara Sandhoefer     : DKH picture change effects
   Avijit Sen             : IP-ROCIS
   Kantharuban Sivalingam : CASSCF convergence, NEVPT2, FIC-MRCI
   Bernardo de Souza      : ESD, SOC TD-DFT
   Georgi Stoychev        : AutoAux, RI-MP2 NMR, DLPNO-MP2 response
   Willem Van den Heuvel  : Paramagnetic NMR
   Boris Wezisla          : Elementary symmetry handling
   Frank Wennmohs         : Technical directorship


 We gratefully acknowledge several colleagues who have allowed us to
 interface, adapt or use parts of their codes:
   Stefan Grimme, W. Hujo, H. Kruse, P. Pracht,  : VdW corrections, initial TS optimization,
                  C. Bannwarth, S. Ehlert          DFT functionals, gCP, sTDA/sTD-DF
   Ed Valeev, F. Pavosevic, A. Kumar             : LibInt (2-el integral package), F12 methods
   Garnet Chan, S. Sharma, J. Yang, R. Olivares  : DMRG
   Ulf Ekstrom                                   : XCFun DFT Library
   Mihaly Kallay                                 : mrcc  (arbitrary order and MRCC methods)
   Jiri Pittner, Ondrej Demel                    : Mk-CCSD
   Frank Weinhold                                : gennbo (NPA and NBO analysis)
   Christopher J. Cramer and Donald G. Truhlar   : smd solvation model
   Lars Goerigk                                  : TD-DFT with DH, B97 family of functionals
   V. Asgeirsson, H. Jonsson                     : NEB implementation
   FAccTs GmbH                                   : IRC, NEB, NEB-TS, DLPNO-Multilevel, CI-OPT
                                                   MM, QMMM, 2- and 3-layer-ONIOM, Crystal-QMMM,
                                                   LR-CPCM, SF, NACMEs, symmetry and pop. for TD-DFT,
                                                   nearIR, NL-DFT gradient (VV10), updates on ESD,
                                                   ML-optimized integration grids
   S Lehtola, MJT Oliveira, MAL Marques          : LibXC Library
   Liviu Ungur et al                             : ANISO software


 Your calculation uses the libint2 library for the computation of 2-el integrals
 For citations please refer to: http://libint.valeyev.net

 Your ORCA version has been built with support for libXC version: 5.1.0
 For citations please refer to: https://tddft.org/programs/libxc/

 This ORCA versions uses:
   CBLAS   interface :  Fast vector & matrix operations
   LAPACKE interface :  Fast linear algebra routines
   SCALAPACK package :  Parallel linear algebra routines
   Shared memory     :  Shared parallel matrices
   BLAS/LAPACK       :  OpenBLAS 0.3.15  USE64BITINT DYNAMIC_ARCH NO_AFFINITY Zen SINGLE_THREADED
        Core in use  :  Zen
   Copyright (c) 2011-2014, The OpenBLAS Project




***************************************
The coordinates will be read from file: water.xyz
***************************************


================================================================================

----- Orbital basis set information -----
Your calculation utilizes the basis: def2-SVP
   F. Weigend and R. Ahlrichs, Phys. Chem. Chem. Phys. 7, 3297 (2005).

----- AuxJ basis set information -----
Your calculation utilizes the auxiliary basis: def2/J
   F. Weigend, Phys. Chem. Chem. Phys. 8, 1057 (2006).

================================================================================
                                        WARNINGS
                       Please study these warnings very carefully!
================================================================================



WARNING: Found flag for RIJCOSX approximation but no HF exchange needed!
  ===> : RIJCOSX  will be disabled and Split-RI-J is turned on

INFO   : the flag for use of the SHARK integral package has been found!

================================================================================
                                       INPUT FILE
================================================================================
NAME = water_spe_vac.inp
|  1> %pal
|  2>   nprocs 4
|  3> end
|  4> 
|  5> %maxcore 350
|  6> 
|  7> ! PBE def2-SVP 
|  8> ! RIJCOSX def2/J
|  9> 
| 10> * xyzfile 0 1 water.xyz
| 11> 
| 12> 
| 13>                          ****END OF INPUT****
================================================================================

                       ****************************
                       * Single Point Calculation *
                       ****************************

---------------------------------
CARTESIAN COORDINATES (ANGSTROEM)
---------------------------------
  O     -3.210350   -0.585040   -0.013950
  H     -2.242470   -0.618270    0.018480
  H     -3.489200   -1.249110    0.634290

----------------------------
CARTESIAN COORDINATES (A.U.)
----------------------------
  NO LB      ZA    FRAG     MASS         X           Y           Z
   0 O     8.0000    0    15.999   -6.066682   -1.105565   -0.026362
   1 H     1.0000    0     1.008   -4.237654   -1.168361    0.034922
   2 H     1.0000    0     1.008   -6.593632   -2.360476    1.198634

--------------------------------
INTERNAL COORDINATES (ANGSTROEM)
--------------------------------
 O      0   0   0     0.000000000000     0.00000000     0.00000000
 H      1   0   0     0.968993102246     0.00000000     0.00000000
 H      1   2   0     0.969000198658   103.97800340     0.00000000

---------------------------
INTERNAL COORDINATES (A.U.)
---------------------------
 O      0   0   0     0.000000000000     0.00000000     0.00000000
 H      1   0   0     1.831131588903     0.00000000     0.00000000
 H      1   2   0     1.831144999180   103.97800340     0.00000000

---------------------
BASIS SET INFORMATION
---------------------
There are 2 groups of distinct atoms

 Group   1 Type O   : 7s4p1d contracted to 3s2p1d pattern {511/31/1}
 Group   2 Type H   : 4s1p contracted to 2s1p pattern {31/1}

Atom   0O    basis set group =>   1
Atom   1H    basis set group =>   2
Atom   2H    basis set group =>   2
---------------------------------
AUXILIARY/J BASIS SET INFORMATION
---------------------------------
There are 2 groups of distinct atoms

 Group   1 Type O   : 12s5p4d2f1g contracted to 6s4p3d1f1g pattern {711111/2111/211/2/1}
 Group   2 Type H   : 5s2p1d contracted to 3s1p1d pattern {311/2/1}

Atom   0O    basis set group =>   1
Atom   1H    basis set group =>   2
Atom   2H    basis set group =>   2


           ************************************************************
           *        Program running with 4 parallel MPI-processes     *
           *              working on a common directory               *
           ************************************************************
------------------------------------------------------------------------------
                           ORCA GTO INTEGRAL CALCULATION
                           -- RI-GTO INTEGRALS CHOSEN --
------------------------------------------------------------------------------
------------------------------------------------------------------------------
                   ___                                                        
                  /   \      - P O W E R E D   B Y -                         
                 /     \                                                     
                 |  |  |   _    _      __       _____    __    __             
                 |  |  |  | |  | |    /  \     |  _  \  |  |  /  |          
                  \  \/   | |  | |   /    \    | | | |  |  | /  /          
                 / \  \   | |__| |  /  /\  \   | |_| |  |  |/  /          
                |  |  |   |  __  | /  /__\  \  |    /   |      \           
                |  |  |   | |  | | |   __   |  |    \   |  |\   \          
                \     /   | |  | | |  |  |  |  | |\  \  |  | \   \       
                 \___/    |_|  |_| |__|  |__|  |_| \__\ |__|  \__/        
                                                                              
                      - O R C A' S   B I G   F R I E N D -                    
                                      &                                       
                       - I N T E G R A L  F E E D E R -                       
                                                                              
 v1 FN, 2020, v2 2021                                                         
------------------------------------------------------------------------------


Reading SHARK input file water_spe_vac.SHARKINP.tmp ... ok
----------------------
SHARK INTEGRAL PACKAGE
----------------------

Number of atoms                             ...      3
Number of basis functions                   ...     24
Number of shells                            ...     12
Maximum angular momentum                    ...      2
Integral batch strategy                     ... SHARK/LIBINT Hybrid
RI-J (if used) integral strategy            ... SPLIT-RIJ (Revised 2003 algorithm where possible)
Printlevel                                  ...      1
Contraction scheme used                     ... SEGMENTED contraction
Coulomb Range Separation                    ... NOT USED
Exchange Range Separation                   ... NOT USED
Finite Nucleus Model                        ... NOT USED
Auxiliary Coulomb fitting basis             ... AVAILABLE
   # of basis functions in Aux-J            ...     71
   # of shells in Aux-J                     ...     25
   Maximum angular momentum in Aux-J        ...      4
Auxiliary J/K fitting basis                 ... NOT available
Auxiliary Correlation fitting basis         ... NOT available
Auxiliary 'external' fitting basis          ... NOT available
Integral threshold                          ...     1.000000e-10
Primitive cut-off                           ...     1.000000e-11
Primitive pair pre-selection threshold      ...     1.000000e-11

Calculating pre-screening integrals         ... done (  0.0 sec) Dimension = 12
Organizing shell pair data                  ... done (  0.0 sec)
Shell pair information
Total number of shell pairs                 ...        78
Shell pairs after pre-screening             ...        78
Total number of primitive shell pairs       ...       272
Primitive shell pairs kept                  ...       265
          la=0 lb=0:     28 shell pairs
          la=1 lb=0:     28 shell pairs
          la=1 lb=1:     10 shell pairs
          la=2 lb=0:      7 shell pairs
          la=2 lb=1:      4 shell pairs
          la=2 lb=2:      1 shell pairs

Calculating one electron integrals          ... done (  0.1 sec)
Calculating RI/J V-Matrix + Cholesky decomp.... done (  0.2 sec)
Calculating Nuclear repulsion               ... done (  0.0 sec) ENN=      9.084297099237 Eh

SHARK setup successfully completed in   0.7 seconds

Maximum memory used throughout the entire GTOINT-calculation: 8.7 MB


           ************************************************************
           *        Program running with 4 parallel MPI-processes     *
           *              working on a common directory               *
           ************************************************************
-------------------------------------------------------------------------------
                                 ORCA SCF
-------------------------------------------------------------------------------

------------
SCF SETTINGS
------------
Hamiltonian:
 Density Functional     Method          .... DFT(GTOs)
 Exchange Functional    Exchange        .... PBE
   PBE kappa parameter   XKappa         ....  0.804000
   PBE mue parameter    XMuePBE         ....  0.219520
 Correlation Functional Correlation     .... PBE
   PBE beta parameter  CBetaPBE         ....  0.066725
 LDA part of GGA corr.  LDAOpt          .... PW91-LDA
 Gradients option       PostSCFGGA      .... off
   Density functional embedding theory  .... OFF
   NL short-range parameter             ....  6.400000
 RI-approximation to the Coulomb term is turned on
   Number of AuxJ basis functions       .... 71


General Settings:
 Integral files         IntName         .... water_spe_vac
 Hartree-Fock type      HFTyp           .... RHF
 Total Charge           Charge          ....    0
 Multiplicity           Mult            ....    1
 Number of Electrons    NEL             ....   10
 Basis Dimension        Dim             ....   24
 Nuclear Repulsion      ENuc            ....      9.0842970992 Eh

Convergence Acceleration:
 DIIS                   CNVDIIS         .... on
   Start iteration      DIISMaxIt       ....    12
   Startup error        DIISStart       ....  0.200000
   # of expansion vecs  DIISMaxEq       ....     5
   Bias factor          DIISBfac        ....   1.050
   Max. coefficient     DIISMaxC        ....  10.000
 Trust-Rad. Augm. Hess. CNVTRAH         .... auto
   Auto Start mean grad. ratio tolernc. ....  1.125000
   Auto Start start iteration           ....    20
   Auto Start num. interpolation iter.  ....    10
   Max. Number of Micro iterations      ....    16
   Max. Number of Macro iterations      .... Maxiter - #DIIS iter
   Number of Davidson start vectors     ....     2
   Converg. threshold I  (grad. norm)   ....   5.000e-05
   Converg. threshold II (energy diff.) ....   1.000e-06
   Grad. Scal. Fac. for Micro threshold ....   0.100
   Minimum threshold for Micro iter.    ....   0.010
   NR start threshold (gradient norm)   ....   0.001
   Initial trust radius                 ....   0.400
   Minimum AH scaling param. (alpha)    ....   1.000
   Maximum AH scaling param. (alpha)    .... 1000.000
   Orbital update algorithm             .... Taylor
   White noise on init. David. guess    .... on
   Maximum white noise                  ....   0.010
   Quad. conv. algorithm                .... NR
 SOSCF                  CNVSOSCF        .... on
   Start iteration      SOSCFMaxIt      ....   150
   Startup grad/error   SOSCFStart      ....  0.003300
 Level Shifting         CNVShift        .... on
   Level shift para.    LevelShift      ....    0.2500
   Turn off err/grad.   ShiftErr        ....    0.0010
 Zerner damping         CNVZerner       .... off
 Static damping         CNVDamp         .... on
   Fraction old density DampFac         ....    0.7000
   Max. Damping (<1)    DampMax         ....    0.9800
   Min. Damping (>=0)   DampMin         ....    0.0000
   Turn off err/grad.   DampErr         ....    0.1000
 Fernandez-Rico         CNVRico         .... off

SCF Procedure:
 Maximum # iterations   MaxIter         ....   125
 SCF integral mode      SCFMode         .... Direct
   Integral package                     .... SHARK and LIBINT hybrid scheme
 Reset frequency        DirectResetFreq ....    20
 Integral Threshold     Thresh          ....  1.000e-10 Eh
 Primitive CutOff       TCut            ....  1.000e-11 Eh

Convergence Tolerance:
 Convergence Check Mode ConvCheckMode   .... Total+1el-Energy
 Convergence forced     ConvForced      .... 0
 Energy Change          TolE            ....  1.000e-06 Eh
 1-El. energy change                    ....  1.000e-03 Eh
 Orbital Gradient       TolG            ....  5.000e-05
 Orbital Rotation angle TolX            ....  5.000e-05
 DIIS Error             TolErr          ....  1.000e-06


Diagonalization of the overlap matrix:
Smallest eigenvalue                        ... 3.830e-02
Time for diagonalization                   ...    0.128 sec
Threshold for overlap eigenvalues          ... 1.000e-08
Number of eigenvalues below threshold      ... 0
Time for construction of square roots      ...    0.009 sec
Total time needed                          ...    0.142 sec

Time for model grid setup =    0.063 sec

------------------------------
INITIAL GUESS: MODEL POTENTIAL
------------------------------
Loading Hartree-Fock densities                     ... done
Calculating cut-offs                               ... done
Initializing the effective Hamiltonian             ... done
Setting up the integral package (SHARK)            ... done
Starting the Coulomb interaction                   ... done (   0.2 sec)
Reading the grid                                   ... done
Mapping shells                                     ... done
Starting the XC term evaluation                    ... done (   0.1 sec)
  promolecular density results
     # of electrons  =      9.998906425
     EX              =     -8.726226855
     EC              =     -0.315781802
     EX+EC           =     -9.042008657
Transforming the Hamiltonian                       ... done (   0.0 sec)
Diagonalizing the Hamiltonian                      ... done (   0.0 sec)
Back transforming the eigenvectors                 ... done (   0.0 sec)
Now organizing SCF variables                       ... done
                      ------------------
                      INITIAL GUESS DONE (   0.3 sec)
                      ------------------
-------------------
DFT GRID GENERATION
-------------------

General Integration Accuracy     IntAcc      ... 4.388
Radial Grid Type                 RadialGrid  ... OptM3 with GC (2021)
Angular Grid (max. ang.)         AngularGrid ... 4 (Lebedev-302)
Angular grid pruning method      GridPruning ... 4 (adaptive)
Weight generation scheme         WeightScheme... Becke
Basis function cutoff            BFCut       ... 1.0000e-10
Integration weight cutoff        WCut        ... 1.0000e-14
Angular grids for H and He will be reduced by one unit
Partially contracted basis set               ... off
Rotationally invariant grid construction     ... off

Total number of grid points                  ...    12750
Total number of batches                      ...      201
Average number of points per batch           ...       63
Average number of grid points per atom       ...     4250
Time for grid setup =    0.049 sec

--------------
SCF ITERATIONS
--------------
ITER       Energy         Delta-E        Max-DP      RMS-DP      [F,P]     Damp
               ***  Starting incremental Fock matrix formation  ***
  0    -76.1856336717   0.000000000000 0.08892582  0.00973783  0.2849653 0.7000
  1    -76.2303576136  -0.044723941893 0.04620716  0.00524889  0.1045469 0.7000
                               ***Turning on DIIS***
  2    -76.2430733786  -0.012715765038 0.04125410  0.00518597  0.0123621 0.0000
  3    -76.2685000929  -0.025426714322 0.02501468  0.00292126  0.0619096 0.0000
  4    -76.2724103534  -0.003910260462 0.00587788  0.00073371  0.0100334 0.0000
                      *** Initiating the SOSCF procedure ***
                           *** Shutting down DIIS ***
                      *** Re-Reading the Fockian *** 
                      *** Removing any level shift *** 
ITER      Energy       Delta-E        Grad      Rot      Max-DP    RMS-DP
  5    -76.27255147  -0.0001411128  0.001137  0.001137  0.002414  0.000253
               *** Restarting incremental Fock matrix formation ***
  6    -76.27256136  -0.0000098982  0.000502  0.001510  0.002254  0.000193
  7    -76.27256200  -0.0000006371  0.000464  0.000270  0.000542  0.000055
  8    -76.27256262  -0.0000006158  0.000198  0.000324  0.000354  0.000044
                 **** Energy Check signals convergence ****
              ***Rediagonalizing the Fockian in SOSCF/NRSCF***

               *****************************************************
               *                     SUCCESS                       *
               *           SCF CONVERGED AFTER   9 CYCLES          *
               *****************************************************


----------------
TOTAL SCF ENERGY
----------------

Total Energy       :          -76.27256275 Eh           -2075.48195 eV

Components:
Nuclear Repulsion  :            9.08429710 Eh             247.19629 eV
Electronic Energy  :          -85.35685985 Eh           -2322.67824 eV
One Electron Energy:         -122.88225748 Eh           -3343.79622 eV
Two Electron Energy:           37.52539762 Eh            1021.11798 eV

Virial components:
Potential Energy   :         -152.05550426 Eh           -4137.64062 eV
Kinetic Energy     :           75.78294150 Eh            2062.15868 eV
Virial Ratio       :            2.00646084


DFT components:
N(Alpha)           :        4.999999996927 electrons
N(Beta)            :        4.999999996927 electrons
N(Total)           :        9.999999993854 electrons
E(X)               :       -8.927502835654 Eh       
E(C)               :       -0.330463439119 Eh       
E(XC)              :       -9.257966274773 Eh       
DFET-embed. en.    :        0.000000000000 Eh       

---------------
SCF CONVERGENCE
---------------

  Last Energy change         ...   -1.3636e-07  Tolerance :   1.0000e-06
  Last MAX-Density change    ...    9.0702e-05  Tolerance :   1.0000e-05
  Last RMS-Density change    ...    9.6137e-06  Tolerance :   1.0000e-06
  Last Orbital Gradient      ...    2.8156e-05  Tolerance :   5.0000e-05
  Last Orbital Rotation      ...    7.7801e-05  Tolerance :   5.0000e-05

             **** THE GBW FILE WAS UPDATED (water_spe_vac.gbw) ****
             **** DENSITY water_spe_vac.scfp WAS UPDATED ****
             **** ENERGY FILE WAS UPDATED (water_spe_vac.en.tmp) ****
             **** THE GBW FILE WAS UPDATED (water_spe_vac.gbw) ****
             **** DENSITY water_spe_vac.scfp WAS UPDATED ****
----------------
ORBITAL ENERGIES
----------------

  NO   OCC          E(Eh)            E(eV) 
   0   2.0000     -18.736617      -509.8493 
   1   2.0000      -0.887230       -24.1428 
   2   2.0000      -0.456008       -12.4086 
   3   2.0000      -0.305382        -8.3099 
   4   2.0000      -0.227747        -6.1973 
   5   0.0000       0.026933         0.7329 
   6   0.0000       0.103962         2.8289 
   7   0.0000       0.514429        13.9983 
   8   0.0000       0.569587        15.4992 
   9   0.0000       0.868814        23.6416 
  10   0.0000       0.871903        23.7257 
  11   0.0000       0.960026        26.1236 
  12   0.0000       1.044413        28.4199 
  13   0.0000       1.287339        35.0303 
  14   0.0000       1.353539        36.8317 
  15   0.0000       1.489142        40.5216 
  16   0.0000       1.718868        46.7728 
  17   0.0000       2.122991        57.7695 
  18   0.0000       2.168130        58.9978 
  19   0.0000       2.865836        77.9834 
  20   0.0000       2.907247        79.1102 
  21   0.0000       3.087149        84.0056 
  22   0.0000       3.396458        92.4223 
  23   0.0000       3.720165       101.2308 

                    ********************************
                    * MULLIKEN POPULATION ANALYSIS *
                    ********************************

-----------------------
MULLIKEN ATOMIC CHARGES
-----------------------
   0 O :   -0.285593
   1 H :    0.142795
   2 H :    0.142798
Sum of atomic charges:    0.0000000

--------------------------------
MULLIKEN REDUCED ORBITAL CHARGES
--------------------------------
  0 O s       :     3.691179  s :     3.691179
      pz      :     1.672095  p :     4.584268
      px      :     1.254201
      py      :     1.657971
      dz2     :     0.001755  d :     0.010146
      dxz     :     0.000745
      dyz     :     0.003134
      dx2y2   :     0.003751
      dxy     :     0.000761
  1 H s       :     0.785910  s :     0.785910
      pz      :     0.018626  p :     0.071294
      px      :     0.034134
      py      :     0.018535
  2 H s       :     0.785908  s :     0.785908
      pz      :     0.026146  p :     0.071293
      px      :     0.018721
      py      :     0.026426


                     *******************************
                     * LOEWDIN POPULATION ANALYSIS *
                     *******************************

----------------------
LOEWDIN ATOMIC CHARGES
----------------------
   0 O :   -0.145351
   1 H :    0.072673
   2 H :    0.072678

-------------------------------
LOEWDIN REDUCED ORBITAL CHARGES
-------------------------------
  0 O s       :     3.442755  s :     3.442755
      pz      :     1.677326  p :     4.683229
      px      :     1.339161
      py      :     1.666742
      dz2     :     0.002857  d :     0.019367
      dxz     :     0.000886
      dyz     :     0.006349
      dx2y2   :     0.008353
      dxy     :     0.000923
  1 H s       :     0.752126  s :     0.752126
      pz      :     0.044200  p :     0.175201
      px      :     0.087289
      py      :     0.043711
  2 H s       :     0.752125  s :     0.752125
      pz      :     0.070547  p :     0.175197
      px      :     0.033290
      py      :     0.071360


                      *****************************
                      * MAYER POPULATION ANALYSIS *
                      *****************************

  NA   - Mulliken gross atomic population
  ZA   - Total nuclear charge
  QA   - Mulliken gross atomic charge
  VA   - Mayer's total valence
  BVA  - Mayer's bonded valence
  FA   - Mayer's free valence

  ATOM       NA         ZA         QA         VA         BVA        FA
  0 O      8.2856     8.0000    -0.2856     2.0210     2.0210     0.0000
  1 H      0.8572     1.0000     0.1428     1.0168     1.0168     0.0000
  2 H      0.8572     1.0000     0.1428     1.0168     1.0168     0.0000

  Mayer bond orders larger than 0.100000
B(  0-O ,  1-H ) :   1.0105 B(  0-O ,  2-H ) :   1.0105 

-------
TIMINGS
-------

Total SCF time: 0 days 0 hours 0 min 3 sec 

Total time                  ....       3.033 sec
Sum of individual times     ....       2.417 sec  ( 79.7%)

Fock matrix formation       ....       1.477 sec  ( 48.7%)
  Split-RI-J                ....       0.597 sec  ( 40.4% of F)
  XC integration            ....       0.381 sec  ( 25.8% of F)
    Basis function eval.    ....       0.017 sec  (  4.5% of XC)
    Density eval.           ....       0.016 sec  (  4.1% of XC)
    XC-Functional eval.     ....       0.018 sec  (  4.7% of XC)
    XC-Potential eval.      ....       0.006 sec  (  1.5% of XC)
Diagonalization             ....       0.151 sec  (  5.0%)
Density matrix formation    ....       0.004 sec  (  0.1%)
Population analysis         ....       0.041 sec  (  1.3%)
Initial guess               ....       0.255 sec  (  8.4%)
Orbital Transformation      ....       0.000 sec  (  0.0%)
Orbital Orthonormalization  ....       0.000 sec  (  0.0%)
DIIS solution               ....       0.271 sec  (  8.9%)
SOSCF solution              ....       0.106 sec  (  3.5%)
Grid generation             ....       0.113 sec  (  3.7%)

Maximum memory used throughout the entire SCF-calculation: 6.2 MB

-------------------------   --------------------
FINAL SINGLE POINT ENERGY       -76.272562753586
-------------------------   --------------------


                            ***************************************
                            *     ORCA property calculations      *
                            ***************************************

                                    ---------------------
                                    Active property flags
                                    ---------------------
   (+) Dipole Moment


------------------------------------------------------------------------------
                       ORCA ELECTRIC PROPERTIES CALCULATION
------------------------------------------------------------------------------

Dipole Moment Calculation                       ... on
Quadrupole Moment Calculation                   ... off
Polarizability Calculation                      ... off
GBWName                                         ... water_spe_vac.gbw
Electron density                                ... water_spe_vac.scfp
The origin for moment calculation is the CENTER OF MASS  = (-5.993827, -1.179295  0.045610)

-------------
DIPOLE MOMENT
-------------
                                X             Y             Z
Electronic contribution:     -0.13144       0.13302      -0.12985
Nuclear contribution   :      0.57352      -0.58041       0.56656
                        -----------------------------------------
Total Dipole Moment    :      0.44208      -0.44739       0.43672
                        -----------------------------------------
Magnitude (a.u.)       :      0.76571
Magnitude (Debye)      :      1.94629



--------------------
Rotational spectrum 
--------------------
 
Rotational constants in cm-1:    26.442708    14.345901     9.300255 
Rotational constants in MHz : 792732.440810 430079.286725 278814.632727 

 Dipole components along the rotational axes: 
x,y,z [a.u.] :     0.000010     0.765713     0.000000 
x,y,z [Debye]:     0.000026     1.946288     0.000000 

 

Timings for individual modules:

Sum of individual times         ...        8.708 sec (=   0.145 min)
GTO integral calculation        ...        4.221 sec (=   0.070 min)  48.5 %
SCF iterations                  ...        4.488 sec (=   0.075 min)  51.5 %
                             ****ORCA TERMINATED NORMALLY****
TOTAL RUN TIME: 0 days 0 hours 0 minutes 11 seconds 886 msec
